gpflow/functions.py

```diff
@@ -40,9 +40,12 @@ class Function(Module):
     The base function class.
     To implement a function, write the ``__call__`` method. This takes a
     tensor X and returns a tensor f(X). In accordance with the GPflow
-    standard, each row of X represents one datum, and each row of Y is computed
+    standard, each row of X represents one datum, and each row of f(X) is computed
     independently for each row of X.
 
+    This class is not only used for mean functions.  For example, the variance of a
+    heteroskedastic model is specified using a Function.
```

Comment:
This is only true for _some_ heteroskedastic models, when the variance is given in a parametric form. You can also build heteroskedastic models where the variance is modeled with another GP. Then it doesn't use this class.

Reply:
```suggestion
    heteroskedastic model can be specified using a `Function`.
```
and add a link to the corresponding notebook?